### ISSUE TITLE
Standard PayPal gateway fails if PayPal is set to only accept encrypted payments

### ISSUE THREAD

**Issue by evohealthse** [woocommerce/woocommerce#19843]
## Prerequisites

- [ X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [ X] The issue still exists against the latest `master` branch of WooCommerce on Github (this is **not** the same version as on WordPress.org!)
- [X ] I have attempted to find the simplest possible steps to reproduce the issue
- [ ] I have included a failing test as a pull request (Optional)

## Steps to reproduce the issue

1. Payment fails with included Wooc Paypal gateway when setting Paypals selling options to only accept encrypted payments. Wooc PayPal Express Checkout still work though. Would be nice If the standard included gateway also was encrypted all the way.
2.
3.

## Expected/actual behavior

When I follow those steps, I see...

I was expecting to see...

## Isolating the problem

- [ X] This bug happens with only WooCommerce plugin active
- [X ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/)
- [ X] I can reproduce this bug consistently using the steps above

## WordPress Environment

<details>
```
`
### WordPress Environment ###

`
### WordPress Environment ###

Home URL: https://shop.evohealth.se
Site URL: https://shop.evohealth.se
WC Version: 3.3.5
Log Directory Writable: ✔
WP Version: 4.9.5
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: sv_SE

### Server Environment ###

Server Info: Apache/2
PHP Version: 7.2.4
PHP Post Max Size: 8 MB
PHP Time Limit: 1200
PHP Max Input Vars: 1000
cURL Version: 7.54.0
OpenSSL/1.0.2k

SUHOSIN Installed: –
Max Upload Size: 8 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.3.5
WC Database Prefix: wp_
MaxMind GeoIP Database: ✔
Total databasstorlek: 2.79MB
Datastorlek för databas: 2.03MB
Indexstorlek för databas: 0.76MB
wp_woocommerce_sessions: Data: 0.11 MB + Index: 0.02 MB
wp_woocommerce_api_keys: Data: 0.02 MB + Index: 0.03 MB
wp_woocommerce_attribute_taxonomies: Data: 0.02 MB + Index: 0.02 MB
wp_woocommerce_downloadable_product_permissions: Data: 0.02 MB + Index: 0.05 MB
wp_woocommerce_order_items: Data: 0.02 MB + Index: 0.02 MB
wp_woocommerce_order_itemmeta: Data: 0.08 MB + Index: 0.03 MB
wp_woocommerce_tax_rates: Data: 0.02 MB + Index: 0.06 MB
wp_woocommerce_tax_rate_locations: Data: 0.02 MB + Index: 0.03 MB
wp_woocommerce_shipping_zones: Data: 0.02 MB + Index: 0.00 MB
wp_woocommerce_shipping_zone_locations: Data: 0.02 MB + Index: 0.03 MB
wp_woocommerce_shipping_zone_methods: Data: 0.02 MB + Index: 0.00 MB
wp_woocommerce_payment_tokens: Data: 0.02 MB + Index: 0.02 MB
wp_woocommerce_payment_tokenmeta: Data: 0.02 MB + Index: 0.03 MB
wp_woocommerce_log: Data: 0.02 MB + Index: 0.02 MB
wp_commentmeta: Data: 0.00 MB + Index: 0.00 MB
wp_comments: Data: 0.00 MB + Index: 0.01 MB
wp_gglcptch_whitelist: Data: 0.02 MB + Index: 0.00 MB
wp_links: Data: 0.00 MB + Index: 0.00 MB
wp_maxbuttonsv3: Data: 0.02 MB + Index: 0.00 MB
wp_maxbuttons_collections: Data: 0.02 MB + Index: 0.00 MB
wp_maxbuttons_collections_trans: Data: 0.02 MB + Index: 0.00 MB
wp_options: Data: 0.98 MB + Index: 0.12 MB
wp_postmeta: Data: 0.26 MB + Index: 0.11 MB
wp_posts: Data: 0.17 MB + Index: 0.04 MB
wp_snippets: Data: 0.02 MB + Index: 0.00 MB
wp_termmeta: Data: 0.02 MB + Index: 0.03 MB
wp_terms: Data: 0.00 MB + Index: 0.01 MB
wp_term_relationships: Data: 0.00 MB + Index: 0.01 MB
wp_term_taxonomy: Data: 0.00 MB + Index: 0.00 MB
wp_usermeta: Data: 0.01 MB + Index: 0.01 MB
wp_users: Data: 0.00 MB + Index: 0.01 MB
wp_wc_download_log: Data: 0.02 MB + Index: 0.03 MB
wp_wc_webhooks: Data: 0.02 MB + Index: 0.02 MB
wp_wonderplugin_gridgallery: Data: 0.02 MB + Index: 0.00 MB

### Post Type Counts ###

attachment: 61
custom_css: 1
customize_changeset: 19
jetpack_migration: 2
jp_img_sitemap: 1
jp_sitemap: 1
jp_sitemap_master: 1
mc4wp-form: 1
mg_items: 9
nav_menu_item: 11
page: 15
post: 4
product: 1
product_variation: 24
revision: 54
robo_gallery_table: 5
shop_coupon: 5
shop_order: 18
shop_order_refund: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (23) ###

BJ Lazy Load: av Bjørn Johansen
Aron Tornberg
angrycreative – 1.0.9

Code Snippets: av Shea Bunge – 2.10.1.1
Google Analytics för WordPress av MonsterInsights: av MonsterInsights – 7.0.5
Google Captcha (reCAPTCHA) by BestWebSoft: av BestWebSoft – 1.36
Jetpack från WordPress.com: av Automattic – 6.0
Lazy Load for Videos: av Kevin Weber – 2.3.1
MailChimp for WordPress: av ibericode – 4.2.1
MaxButtons: av Max Foundry – 7.1
MailChimp for WordPress - Captcha: av ibericode – 1.1.2
Media Grid: av Luca Montanari – 6.1
Media Library Assistant: av David Lingren
Fair Trade Judaica – 2.72

Really Simple SSL: av Rogier Lankhorst – 2.5.26
Remove "Designed By..." Footer from Storefront: av Daniel Santoro – 1.0.0
Robo Gallery: av RoboSoft – 2.7.9
Robo Gallery Key: av WpTools – 1.5
Storefront Sticky Add to Cart: av WooThemes – 1.1.8
WonderPlugin Portfolio Grid Gallery: av Magic Hills Pty Ltd – 12.3
Pacsoft/Unifaun integration for WooCommerce: av Wetail – 2.3.17 – Inte testat med den aktiva versionen för WooCommerce
WooCommerce Visma integration: av Wetail – 1.0 – Inte testat med den aktiva versionen för WooCommerce
WooCommerce Cart Tab: av jameskoster – 1.1.1 – Inte testat med den aktiva versionen för WooCommerce
WooCommerce PayPal Express Checkout Gateway: av WooCommerce – 1.5.3 – Inte testat med den aktiva versionen för WooCommerce
WooCommerce Services: av Automattic – 1.12.3 – Inte testat med den aktiva versionen för WooCommerce
WooCommerce: av Automattic – 3.3.5

### Settings ###

API Enabled: ✔
Force SSL: ✔
Currency: SEK (kr)
Currency Position: right_space
Thousand Separator: 
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Butikbas: #4 - /butik/
Varukorg: #5 - /varukorg/
Kassan: #6 - /kassan/
Mitt konto: #7 - /mitt-konto/
Allmänna villkor: #26 - /kopvillkor/

### Theme ###

Name: Storefront
Version: 2.2.8
Author URL: https://woocommerce.com/
Child Theme: ❌ – Om du modifierar WooCommerce på ett modertema som du själv inte byggt
så rekommenderar vi att du använder ett barntema istället. Läs: Hur du skapar ett barntema

WooCommerce Support: ✔

### Templates ###

Overrides: –
`
```
</details>

**Comment by mikejolley** [comment]
Similar issue raised here https://www.paypal-community.com/t5/Button-APIs/Cannot-create-ENCRYPTED-CART-UPLOAD-button-but-unsecured-works/td-p/1267264

tl;dr Because the cart is dynamic you cannot have a pre-made encrypted button. Security is implemented by checking the actual payment against the IPN requests to ensure it wasn't tampered with. What you're requesting is not possible for PayPal standard.
